setup.cfg

```diff
@@ -1,21 +0,0 @@
-[pep8]
```

Comment:
I'm warming to the idea of using this tool, but I think that a distinction between linting and formatting should be maintained.  We should probably not get rid of pep8 and flake8.  What will Zuul be doing, after all?

Reply:
We *could* make a different target that uses black for CI, e.g. running `black --check`

```
(set -o pipefail && black --skip-string-normalization --check --fast --line-length 160 awx awxkit awx_collection | tee reports/black.report)
would reformat /home/ryanpetrello/dev/awx/awx/main/tasks.py
Oh no! 💥 💔 💥
1 file would be reformatted, 809 files would be left unchanged.
make: *** [Makefile:275: black] Error 1
~/dev/awx echo $?
2
```

Reply:
```tox -e linters
GLOB sdist-make: /home/ryanpetrello/dev/awx/setup.py
linters inst-nodeps: /home/ryanpetrello/dev/awx/.tox/.tmp/package/1/awx-17.0.1.zip
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
linters installed: -f file:///Users/ryan/.pip/wheelhouse,appdirs==1.4.4,arrow==1.0.3,awx==17.0.1,black==20.8b1,click==7.1.2,Jinja2==2.11.3,jinja2-time==0.2.0,make==0.1.6.post2,MarkupSafe==1.1.1,mypy-extensions==0.4.3,pathspec==0.8.1,python-dateutil==2.8.1,PyYAML==5.4.1,regex==2021.3.17,six==1.15.0,toml==0.10.2,typed-ast==1.4.2,typing-extensions==3.7.4.3,yamllint==1.26.0
linters run-test-pre: PYTHONHASHSEED='1240567368'
linters run-test: commands[0] | make black --check
(set -o pipefail && black --skip-string-normalization --check --fast --line-length 160 awx awxkit awx_collection | tee reports/black.report)
would reformat /home/ryanpetrello/dev/awx/awx/main/tasks.py
Oh no! 💥 💔 💥
1 file would be reformatted, 809 files would be left unchanged.
make: *** [Makefile:275: black] Error 1
ERROR: InvocationError for command /usr/bin/make black --check (exited with code 2)```